src/lib.rs

```diff
@@ -161,78 +161,128 @@ pub use multihash;
 pub use tokio_io;
 pub use tokio_codec;
 
-#[doc(inline)]
-pub use libp2p_core as core;
-#[cfg(not(any(target_os = "emscripten", target_os = "unknown")))]
-#[doc(inline)]
-pub use libp2p_deflate as deflate;
-#[cfg(not(any(target_os = "emscripten", target_os = "unknown")))]
-#[doc(inline)]
-pub use libp2p_dns as dns;
-#[doc(inline)]
-pub use libp2p_identify as identify;
-#[doc(inline)]
-pub use libp2p_kad as kad;
-#[doc(inline)]
-pub use libp2p_floodsub as floodsub;
-#[doc(inline)]
-pub use libp2p_mplex as mplex;
-#[cfg(not(any(target_os = "emscripten", target_os = "unknown")))]
-#[doc(inline)]
-pub use libp2p_mdns as mdns;
-#[cfg(not(any(target_os = "emscripten", target_os = "unknown")))]
-#[doc(inline)]
-pub use libp2p_noise as noise;
-#[doc(inline)]
-pub use libp2p_ping as ping;
-#[doc(inline)]
-pub use libp2p_plaintext as plaintext;
-#[doc(inline)]
-pub use libp2p_ratelimit as ratelimit;
-#[doc(inline)]
-pub use libp2p_secio as secio;
+/// We re-export libp2p-core for the `NetworkBehaviour` macro.
+#[doc(hidden)]
+pub use libp2p_core;
```

Comment:
```suggestion
pub use libp2p_core as core;
```

Reply:
Note that it's a hidden re-export (because of the procedural macro). Should I change that anyway?

Reply:
Sorry, missed that. Please discard the suggestion.